src/renderers/shaders/ShaderChunk/envmap_common_pars_fragment.glsl.js

```diff
@@ -1,6 +1,7 @@
 export default /* glsl */`
 #ifdef USE_ENVMAP
 
+	uniform float backgroundBlurriness;
```

Comment:
It should be clear now that this does not belong here.

The background needs its own shader chunks.

I hope we can agree that a coordinated effort at refactoring is in order.

Reply:
In a later, PR, of course.